test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
+1 on killing Marionette.Callbacks - i wrote those before I knew what jQuery.Deffered and Promises were.

-1 for killing this behavior in the Application object. I use this feature, still. There are a lot of systems out there, that use it. 

that doesn't mean you can't get rid of it, but if you destroy this behavior, what will you recommend people do to replace it?